src/main/resources/application.yml

```diff
@@ -80,6 +85,7 @@ staff-notifications:
 feature_toggles:
   watermark_pdf: false
   document_management: false
+  core_case_data: true
```

Comment:
I would prefer to have CCD disabled by default.

Reply:
done